### ISSUE TITLE
Run all test files in a folder

### ISSUE THREAD

**Issue by dialex** [mobile-dev-inc/maestro#335]
If I try to run `maestro test test/folder` then I get an error saying that is a directory.

Same error if I do `maestro test test/**/*.yml`.

Can you please support a passing a directory and then Maestro runs all the `*.yml` files there, maybe in alphabetical order? That's pretty standard nowadays.

Thank you!

**Comment by dialex** [comment]
This is a duplicate of https://github.com/mobile-dev-inc/maestro/issues/311

**Comment by dialex** [comment]
Wait `maestro test test/*.yml` actually works! That's good enough for now

**Comment by dialex** [comment]
Actually it can't. I get another error `Unmatched argument at index 2:`, so even tho that *.yml is accepted, as soon as there are 2 or more files in that folder the run cmd will fail 😞

**Comment by dialex** [comment]
It's an ugly workaround but it works. Please fix this properly:

```sh
for i in test/e2e/*.yml; do maestro test $i; done
```

```js
// package.json
"e2e:maestro": "for i in test/e2e/*.yml; do maestro test $i; done",
```

**Comment by qajonatasmartins** [comment]
i have the same problem. I need to run all test files.
`$ maestro test -e NAME_APP=org.wikimedia.wikipedia ./tests/**/*.yaml
Unmatched argument at index 4: './tests/onboarding/onboarding-step-two.yaml'
Usage: maestro test [-c] [-e=<String=String>]... <flowFile>
      <flowFile>
  -c, --continuous
  -e, --env=<String=String>`

**Comment by dmitry-zaitsev** [comment]
This was addressed as part of #397 , specifying a folder name instead of a file name will run all the tests in a folder (excluding subfolders).

This change is going to be a part of 1.15.0 planned for this week (or early next week)
